keyboards/santy65/info.json

```diff
@@ -0,0 +1,134 @@
+{
+    "manufacturer": "Santy",
+    "keyboard_name": "santy65",
+    "maintainer": "qmk",
+    "bootloader": "qmk-dfu",
+    "build": {
+        "lto": true
+    },
+    "diode_direction": "COL2ROW",
+    "encoder": {
+        "rotary": [{ "pin_a": "B6", "pin_b": "C7" }]
+    },
+    "features": {
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "encoder": true,
+        "encoder_map": true,
+        "pointing_device": true
+    },
+    "matrix_pins": {
+        "cols": ["B4", "D5", "F4", "F5", "D4", "C6", "D7", "E6", "D1"],
+        "rows": ["F6", "F7", "F0", "F1", "B5"]
+    },
+    "processor": "atmega32u4",
+    "split": {
+        "enabled": true,
+        "encoder": {
+            "right": {
+                "rotary": [{ "pin_a": "B6", "pin_b": "B2" }]
+            }
+        },
+        "matrix_pins": {
+            "right": {
+                "cols": ["B4", "D2", "F4", "F5", "D4", "C6", "D7", "E6", "D1"],
+                "rows": ["F6", "F7", "B1", "B3", "B5"]
+            }
+        },
+        "soft_serial_pin": "D0",
+        "usb_detect": {
+            "enabled": true,
+            "polling_interval": 10,
+            "timeout": 2000
+        }
+    },
+    "usb": {
+        "device_version": "0.0.1",
+        "force_nkro": true,
+        "pid": "0x0001",
+        "vid": "0x0001"
+    },
+    "layouts": {
+        "LAYOUT": {
+            "layout": [
+                { "label": "L00", "matrix": [0, 0], "x": 0, "y": 0 },
```

Comment:
The x/y coordinates here do not look correct, and should be fixed to display the layout appropriately. You can use `qmk info -l -kb santy65` or upload it to the QMK Configurator using ctrl + shift + i.

Reply:
I fixed the coordinates so that it looks like the keyboard layout on the configurator as well as when you use the qmk info -l -kb santy65 command